create/remote-pipelines-tasks-resolvers.adoc

```diff
@@ -36,50 +44,52 @@ include::modules/op-resolver-hub-config.adoc[leveloffset=+2]
 include::modules/op-resolver-hub.adoc[leveloffset=+2]
 
 [id="resolver-bundles_{context}"]
-== Specifying a remote pipeline or task from a Tekton bundle
+== Specifying a remote pipeline, task, or step action from a Tekton bundle
 
-You can use the bundles resolver to specify a remote pipeline or task from a Tekton bundle. A Tekton bundle is an OCI image available from any OCI repository, such as an OpenShift container repository.
+You can use the bundles resolver to specify a remote pipeline, task, or `StepAction` definition from a Tekton bundle. A Tekton bundle is an OCI image available from any OCI repository, such as an OpenShift container repository.
 
 include::modules/op-resolver-bundle-config.adoc[leveloffset=+2]
 include::modules/op-resolver-bundle.adoc[leveloffset=+2]
 
-[id="resolver-cluster_{context}"]
-== Specifying a pipeline or task from the same cluster
-
-You can use the cluster resolver to specify a pipeline or task that is defined in a namespace on the {OCP} cluster where {pipelines-title} is running.
-
-In particular, you can use the cluster resolver to access tasks that {pipelines-shortname} provides in its installation namespace, which is normally the `openshift-pipelines` namespace.
-
-include::modules/op-resolver-cluster-config.adoc[leveloffset=+2]
-include::modules/op-resolver-cluster.adoc[leveloffset=+2]
-
-include::modules/op-resolver-cluster-tasks-ref.adoc[leveloffset=+1]
-
 [id="resolver-git-anon_{context}"]
-== Specifying a remote pipeline or task with anonymous Git cloning
+== Specifying a remote pipeline, task, or step action with anonymous Git cloning
 
-You can use the Git resolver to access a remote pipeline or task from a Git repository. The repository must include a YAML file that defines the pipeline or task. For anonymous access, you can clone repositories with the resolver without needing authentication credentials.
+You can use the Git resolver to access a remote pipeline, task, or `StepAction` definition from a Git repository. The repository must include a YAML file that defines the pipeline or task. For anonymous access, you can clone repositories with the resolver without needing authentication credentials.
 
 include::modules/op-resolver-git-config-anon.adoc[leveloffset=+2]
 include::modules/op-resolver-git-anon.adoc[leveloffset=+2]
 
 [id="resolver-git-scm_{context}"]
-== Specifying a remote pipeline or task with an authenticated API
+== Specifying a remote pipeline, task, or step action with an authenticated Git API
```

Comment:
```suggestion
== Specifying a remote pipeline, task, or StepAction with an authenticated Git API
```

Reply:
The style guide does not permit us to use StepAction as a standalone noun. It can be a StepAction definition, StepAction resource, and so on, but when it's a standalone thing, it's "step action".